Makefile

```diff
@@ -16,15 +16,9 @@ cache = $(if $(cached-$1),,$(eval cached-$1 := 1)$(eval cache-$1 := $($1)))$(cac
 
 # force rita to be rebuilt even if it's up to date
 .PHONY: rita
-rita: vendor $(SRC)
+rita: $(SRC)
 	go build ${LDFLAGS}
 
-vendor: Gopkg.lock
```

Comment:
From what I can tell go's builtin tooling takes care of the vendoring behind the scenes. So just running `go build` will automatically download missing dependencies rather than producing an error. This means our Makefile can be simpler.

Reply:
I updated to go 14.2 before testing this branch, and got a whole bunch of `go: inconsistent vendoring` errors. Do we need to figure out how to correctly transition existing go users to do this update?

Reply:
I think I had the same thing. For me it was due to already having a `vendor` directory. Once I deleted that the issues went away. Do you think it's worth putting a `rm -rf vendor` statement in the makefile to automate this for people (assuming it also fixes it for you)? `go mod vendor` can also create a vendor directory but I don't see us having to use that so it should be safe to just delete it.

Reply:
yes, that sounds like a perfect idea, lets add the removal step.

Reply:
and it did fix it for me, thank you!